models/games.model.js

```diff
@@ -25,4 +25,15 @@ exports.selectReviews = () => {
         .then((data) => {
             return data.rows;
         });
+};
+
+exports.selectCommentsByReviewId = (review_id) => {
+    return db
+        .query(`SELECT * FROM comments WHERE review_id = $1 ORDER BY created_at DESC;`, [review_id])
+        .then((data) => {
+            if (data.rows.length === 0) {
```

Comment:
This error currently will trigger when a valid article has no comments (when really in this case we would just need to return an empty array of comments without an error), we'll need to avoid this by making sure if the article exists